rules.js

```diff
@@ -186,6 +186,103 @@ class sub_with_return extends rule(3) {
     }
 }
 
+class object_formatting extends rule(4) {
+    constructor() {
+        super()
+        this._node = 'object'
+        this.message = ``
+        this.spaces_after_colon = 1
+        this.spaces_before_colon = 0
```

Comment:
consider `whitespace_*` over `spaces_*` 

developer may misconstrue space to mean only ` ` char, when this will check for tabs too

Reply:
Hmm. My intent to only allow spaces and not tabs. Will change code.